tests/keras_autolog/test_keras_autolog.py

```diff
@@ -129,15 +150,15 @@ def test_keras_autolog_logs_expected_data(keras_random_data_run):
     assert data.params['optimizer_name'] == 'Adam'
     assert 'epsilon' in data.params
     assert data.params['epsilon'] == '1e-07'
-    assert 'model_summary' in data.tags
```

Comment:
I would still recommend keeping this line since it's a defensive check that the tag actually exists in `data.tags`.